java/code/src/com/redhat/rhn/frontend/xmlrpc/errata/ErrataHandler.java

```diff
@@ -152,6 +156,15 @@ public Map<String, Object> getDetails(User loggedInUser, String advisoryName) th
         if (errata.getAdvisoryStatus() != null) {
             errataMap.put("advisory_status", errata.getAdvisoryStatus().getMetadataValue());
         }
+
+        try {
+            final VendorSpecificErrataParser parser = ErrataParserFactory.getParser(errata);
+            errataMap.put("vendor_advisory", parser.getAdvisoryUri(errata).toString());
+        }
+        catch (ErrataParsingException ex) {
+            throw new InvalidErrataException("Unable to parse errata vendor");
+        }
```

Comment:
I don't know how this is possible, but maybe we could still continue to output without this data? Is this case expected occasionally?

Reply:
Alternatively we could just `errataMap.put("vendor_advisory", "");`
Spacecmd would then print
```
Vendor Advisory
---------------
N/A
```
in this case

Reply:
I just double checked how we handle it in the webui: We catch the exception and log a info message. So it seems this could potentially happen and isn't really handled as an error.

Reply:
Or I'd say even `debug` level suits better. I don't think any logging is necessary unless we're debugging something about it.